templates/docs/patterns/card.md

```diff
@@ -58,6 +58,32 @@ The purpose of the overlay card is to make the text visible in conjunction with
 View example of the patterns card overlay
 </a></div>
 
+## Accessibility
+
+{# copy doc: https://docs.google.com/document/d/1f_dJS0MquxNm9D6e1OaHohyqXBNG1GwnPfFdxFci60k/edit# #}
```

Comment:
I don't think we want to keep these docs in 2 places, so once they are in code we probably don't need the copy doc.